lab_1_classify_by_unigrams/main.py

```diff
@@ -71,7 +137,13 @@ def load_profile(path_to_file: str) -> dict | None:
     :param path_to_file: a path to the language profile
     :return: a dictionary with at least two keys – name, freq
     """
-
+    if not isinstance(path_to_file, str):
+        return None
+    with open(path_to_file, "r", encoding="utf-8") as file_to_read:
+        language_profile = json.load(file_to_read)
+    if isinstance(language_profile, dict):
+        return language_profile
+    return None
```

Comment:
Basically, this line does not have any impact: if you removed it, the function would behave in the same way. However, I understand that this is to battle pylint (or mypy). This is why it is better to use early returns: if we encounter unsatisfying behaviour, we return `None` immediately, and in some cases we also save identation.

Please change the order of returns here